### DISCUSSION TITLE
Welcome to AllTameable Discussions and Feature Suggestions!

### DISCUSSION THREAD

**Discussion by meldurson** [meldurson/AllTameable#2] [Ideas]
## 👋 Welcome!
  We’re using Discussions as a place to make suggestions on how the mod could be improved or features. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas or features.
  * Be respectful of others.
  * Note, this is not for bugs or errors with the mod. For that message or ping me on [Discord](https://discord.gg/RBq2mzeu4z) under Meldurson.

**Comment by Pluralcake** [comment]
Could it be possible to make it so boars also follow and stay on command like the other creatures?

**Reply by meldurson** [reply]
Yes. In the tamelist, the second term when specifying a creature is if it commandable so
Boar,true
Would make it commandable

**Comment by Kheldar109** [comment]
Is there a connection to DNA attributes and pet Tier level? if not, could there be?

**Comment by hoobio** [comment]
Would it be possible add a feature that allows tames to teleport to the player they are following if they are a certain distance away?
It is [implemented in this example from the ServerSideQoL mod](https://github.com/ArgusMagnus/ValheimServersideQoL/blob/7f16aa57c782ca3eac7c0c9663539beb571516d0/Processors/PlayerProcessor.cs#L79C8-L101C10) however doesn't appear to work for tames where follow state is set by this plugin
